### PR TITLE
remove build --all

### PR DESCRIPTION
**Pull request by rob-solana**
#### Problem
 cargo build --all is unnecessary now that inter-project dependencies are correct

 #### Summary of Changes
 remove build --all

Fixes #

### CODE REVIEW

File: ci/test-stable.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ case $testName in
 test-stable)
   echo "Executing $testName"
 
-  _ cargo +"$rust_stable" build --all ${V:+--verbose}
+#  _ cargo +"$rust_stable" build --all ${V:+--verbose}
```
Comment (mvines) [inline ci/test-stable.sh]: remove it entirely?

**Comment by mvines** [comment]
Hmm, CI has been failing 100% since this PR was merged in the stable build.  But the PR build was green 🤔 

https://buildkite.com/solana-labs/solana/builds?branch=master

**Comment by rob-solana** [comment]
weird, wanna revert?

**Comment by rob-solana** [comment]
I was seeing this error earlier today:
```
test test_repairman_catchup ... thread 'solana-replay-stage' panicked at 'Create directory failed: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:997:5
```

**Comment by mvines** [comment]
Yeah I think so (revert).

The repairman failure is not new,been intermittent for a while now

**Comment by rob-solana** [comment]
looks promising: https://buildkite.com/solana-labs/solana/builds/8084#dd6e0b05-1c75-44e1-864d-4dc204822f28

**Pull request merged**